tools/travis-test.sh

```diff
@@ -28,14 +28,17 @@ if [ -z "$USE_DEBUG" ]; then
   if [ -z "$IN_CHROOT" ]; then
     $PIP install .
   else
-    sysflags="$($PYTHON -c "from distutils import sysconfig; print (sysconfig.get_config_var('CFLAGS'))")"
+    sysflags="$($PYTHON -c "from distutils import sysconfig; \
+        print (sysconfig.get_config_var('CFLAGS'))")"
     CFLAGS="$sysflags $werrors -Wlogical-op" $PIP install . 2>&1 | tee log
-    grep -v "_configtest" log | grep -vE "ld returned 1|no previously-included files matching" | grep -E "warning\>";
-    # accept a mysterious memset warning that shows with -flto
```

Comment:
Probably... Are we using lto in any configurations at this point?
@juliantaylor?